### PR TITLE
release notes: schema tracking no longer experimental

### PR DESCRIPTION
**Pull request by deepthi**
## Description
We forgot to document this in the summary release notes.

## Related Issue(s)
#10595 

## Checklist

-   [x] "Backport me!" label has been added if this change should be backported
-   [x] Tests were added or are not required
-   [x] Documentation was added or is not required https://github.com/vitessio/website/pull/1078

### CODE REVIEW

**Review by systay — APPROVED** [review]

File: doc/releasenotes/14_0_0_summary.md [hunk 1]
```diff
@@ -26,6 +27,11 @@ You can always roll back to the earlier planner, either by providing the flag `-
 select /*vt+ PLANNER=V3 */ name, count(*) from users
 ```
 
+### Schema Tracking is no longer experimental
+
+Schema tracking is now GA. It is enabled by default. Schema tracking allows the Gen4 planner to plan many more queries.
+This can be turned off if necessary (for example, if you choose to continue to use the V3 planner) by changing the [relevant flags](https://vitess.io/docs/14.0/reference/features/schema-tracking/) on VTGate and VTTablet.
```
Comment (harshit-gangal) [inline doc/releasenotes/14_0_0_summary.md:33]: schema tracking is still useful if you use v3 planner. v3 also takes advantage of the authoritative column list.

The reason someone wants to disable is when they see issues with it.

Comment (frouioui) [reply doc/releasenotes/14_0_0_summary.md:33]: Do you have examples of situations where v3 takes advantage of the authoritative column list? One that I can think of is if someone uses the `v3` planner but we discretely plan the query using `Gen4` if for instance there is a LEFT JOIN.

Comment (frouioui) [reply doc/releasenotes/14_0_0_summary.md]: @harshit-gangal, I have removed the mention of `v3` in the sentence.

**Review by systay — APPROVED** [review]

File: doc/releasenotes/14_0_0_release_notes.md [hunk 2]
```diff
@@ -27,6 +28,11 @@ You can always roll back to the earlier planner, either by providing the flag `-
 select /*vt+ PLANNER=V3 */ name, count(*) from users
 ```
 
+### Schema Tracking is no longer experimental
+
+Schema tracking is now GA. It is enabled by default. Schema tracking allows the Gen4 planner to plan many more queries.
+This can be turned off if necessary (for example, if you choose to continue to use the V3 planner) by changing the [relevant flags](https://vitess.io/docs/14.0/reference/features/schema-tracking/) on VTGate and VTTablet.
```
Comment (harshit-gangal) [inline doc/releasenotes/14_0_0_release_notes.md:34]: this also needs correction.

**Review by harshit-gangal — APPROVED** [review]

**Pull request merged**